src/repl.ml

```diff
@@ -20,7 +20,7 @@ let ocaml_utop_setting =
   Settings.create_setting ~scope:Global ~key:"ocaml.useUtop"
     ~of_json:Jsonoo.Decode.bool ~to_json:Jsonoo.Encode.bool
 
-let use_utop () = Settings.get ocaml_utop_setting
+let use_utop = Settings.get ocaml_utop_setting
```

Comment:
You should keep the `()` argument to make it a function. Without the argument, the setting is only read once and the user would have to restart the extension after changing the utop setting.

Reply:
Done